src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+
+    @Override
+    public void doWork() {
+        System.out.println("Truck started its work.");

```

Comment:
The message in the `doWork` method should be more informative and indicate the specific machine type that is working. Consider including the class name in the message, for example: 'Truck is doing its work.'